src/main/java/edu/harvard/lib/librarycloud/collections/CollectionsAPI.java

```diff
@@ -417,14 +426,17 @@ public Response updateCollection(
                                      @PathParam("id") Integer id, Collection collection
                                      ) {
         if (!this.canEditItems(id)) {
-            //return Response.status(Status.UNAUTHORIZED).build();
             throw new LibraryCloudCollectionsException("Not Authorized", Status.UNAUTHORIZED);
 
         }
-      Collection result = collectionDao.updateCollection(id,collection);
+      User user = (User)securityContext.getUserPrincipal();
+      Collection result = collectionDao.updateCollection(id,collection,user);
       if (result != null) {
           try {
               collectionsWorkflow.notify(result);
+              //Respond with the collection json info
+              GenericEntity entity = new GenericEntity<Collection>(result){};
+              return Response.ok(entity).build();
```

Comment:
I'm not certain if this is an issue on my end, but I was not receiving the response that was built here. It seemed to function as expected when I moved the return response to outside the try/catch block (replacing the `return Response.status(Status.NO_CONTENT).build();` on line 432)

Reply:
Changes made and committed